.github/workflows/ci.yml

```diff
@@ -1,59 +1,29 @@
 name: CI
-on:
-  pull_request:
-    branches: [ main ]
-
-# env: # testing if we need this..
-#   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        description: Python version to use in the CI
+        required: false
+        type: string
+        default: '["3.10", "3.11", "3.12"]'
 jobs:
-  lint:
-    uses: AllenNeuralDynamics/aind-github-actions/.github/workflows/lint.yml@main
-    secrets:
-      SERVICE_TOKEN: ${{ secrets.GITHUB_TOKEN }} # required
-
-  tag:
-    uses: AllenNeuralDynamics/aind-github-actions/.github/workflows/tag.yml@main
-    secrets:
-      SERVICE_TOKEN: ${{ secrets.GITHUB_TOKEN }} # required
-
   ci:
     name: Run CI Tasks
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
Subsequent steps reference job matrix, so if you're going to remove this then you need to remove those lines.